libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1298,5 +1323,18 @@ unsigned CompilerUtils::prepareMemoryStore(Type const& _type, bool _padToWords)
 	return numBytes;
 }
 
+void CompilerUtils::convertFixedPointType(unsigned int _from, unsigned int _to, bool _signed)
+{
+	if (_from > _to)
```

Comment:
Oh, this could have an assert for `_from <= 78` and `_to <= 78` (or whatever is the limit).

Reply:
I feel like that violates separation of concerns.

Reply:
There's no guarantee this function gets a value from a range checked previously.